eos/saveddata/fit.py

```diff
@@ -461,11 +461,16 @@ def calculateModifiedAttributes(self, targetFit=None, withBoosters=False, dirtyS
         timer = Timer(u'Fit: {}, {}'.format(self.ID, self.name), logger)
         logger.debug("Starting fit calculation on: %r, withBoosters: %s", self, withBoosters)
 
+        try:
+            projectionInfo = self.getProjectionInfo(targetFit.ID)
+        except:
+            pass
+        else:
+            logger.debug("ProjectionInfo: %s", projectionInfo)
+
```

Comment:
>Or, you could probably set projectionInfo = None outside the if statement, and put the debug log back in the if block. That assigns the variables and prevents pyCharm from yelling whatever obscenities it yells with this kind of 'error'. I would be okay with that, and it looks cleaner. :)

Brilliant! 

Except I already tried this, and then pyCharm complains that it's assigned and not used, because it knows it's a bullshit assignment. ☺

I'll revert it for now, we can look at it later. It's not a major thing.